sycl/include/sycl/group.hpp

```diff
@@ -673,13 +673,7 @@ template <int Dimensions = 1> class __SYCL_TYPE(group) group {
   friend class detail::Builder;
   group(const range<Dimensions> &G, const range<Dimensions> &L,
         const range<Dimensions> GroupRange, const id<Dimensions> &I)
-      : globalRange(G), localRange(L), groupRange(GroupRange), index(I) {
-    // Make sure local range divides global without remainder:
-    __SYCL_ASSERT(((G % L).size() == 0) &&
-                  "global range is not multiple of local");
-    __SYCL_ASSERT((((G / L) - GroupRange).size() == 0) &&
-                  "inconsistent group constructor arguments");
-  }
```

Comment:
For some reason, these changes makes MSVC think that this could potentially cause division by 0. However, this ctor cannot be called on host (since `group` is not usable on host) so these asserts carry no meaning.